bin/cgp.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Imperfect but simple script to pass governance proposals on Celo tesnets (staging, baklava or alfajores)
```

Comment:
update this? Looks like it was taken from the other script. Also maybe add an example usage?